app/src/androidTest/java/org/mozilla/focus/activity/BadURLTest.java

```diff
@@ -48,6 +49,12 @@ protected void beforeActivityLaunched() {
         }
     };
 
+    // Disabled in geckoview since it accesses web element
+    @Before
+    public void checkWebview() {
+        org.junit.Assume.assumeFalse(TestHelper.getAppName().contains("gecko"));
+    }
```

Comment:
Aren't those tests that would make sense to move to the webview test folder for now?

Reply:
I was thinking about leaving them in the common folder and simplify the tests later to work on geckoview, but I guess it's better to put it in a separate directory for now.